.markdownlint.json

```
@@ -0,0 +1,33 @@
+{
+  "extends": "markdownlint/style/prettier",
+  "ignores": [
+    ".claude/**",
+    "tmp/**"
+  ],
+  "rules": {
+    "MD001": true,
```

Biome (2.1.2)

[error] The key MD001 was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)